Microservices/Posts/src/main/java/cs4337/group_8/TemplateProject/controllers/PostController.java

```diff
@@ -18,14 +18,17 @@ public class PostController {
 
     @Autowired
     private PostService postService;
+    private final JwtService jwtService;
```

Comment:
That is not injecting.
Remove the `@Autowired` as its deprecated and use constructor based injection.
```
private PostService postService;
private final JwtService jwtService;

public PostController(PostService postService, JwtService jwtService){
    this.postService = postService;
    this.jwtService = jwtService;
}
```